NEWS.md

```diff
@@ -1,3 +1,3 @@
-# robust.prioritizr 0.0.0.1
```

Comment:
Rather than over-writing the previous version, I'd suggest adding a new entry here for this new version that explains the update (e.g., if you think everything's working now, you could explain this is the first stable release of something, e.g., see https://github.com/tidyverse/dplyr/blob/main/NEWS.md)? I'd also suggest updating the version number if the DESCRIPTION to math (if needed).